policyengine_us/variables/gov/states/nj/tax/income/credits/property_tax_credit/nj_property_tax_credit_eligible.py

```diff
@@ -23,10 +26,8 @@ def formula(tax_unit, period, parameters):
         disabled_head = tax_unit("disabled_head", period)
         blind_spouse = tax_unit("blind_spouse", period)
         disabled_spouse = tax_unit("disabled_spouse", period)
-        senior_head = tax_unit("age_head", period) >= p.senior_qualifying_age
-        senior_spouse = (
-            tax_unit("age_spouse", period) >= p.senior_qualifying_age
-        )
+        senior_head = tax_unit("age_head", period) >= p.age_threshold
```

Comment:
one fewer computation
```suggestion
        senior_head_or_spouse = tax_unit("greater_age_head_spouse", period) >= p.age_threshold
```